comment/tests/base.py

```diff
@@ -411,15 +427,25 @@ def setUpTestData(cls):
         }
         cls.comment = cls.create_comment(cls.post_1, cls.user_1)
 
-    def get_url(self, base_url=None, **kwargs):
+    @classmethod
+    def get_url(cls, base_url=None, **kwargs):
         if not base_url:
-            base_url = self.base_url
+            base_url = cls.base_url
         if kwargs:
             base_url += '?'
             for (key, val) in kwargs.items():
                 base_url += str(key) + '=' + str(val) + '&'
         return base_url.rstrip('&')
 
+    def assert_permission_denied_response(self, response, reason=None):
+        forbidden_code = 403
+        data = json.loads(response.content.decode('utf-8'))
+
+        self.assertEqual(response.status_code, forbidden_code)
+        self.assertEqual(data['status'], forbidden_code)
+        if reason:
+            self.assertEqual(data['reason'], reason)
+
```

Comment:
This method is already defined in the `base class` https://github.com/Radi85/Comment/pull/201/files#diff-c2eb376f4cca55dbd53c16353ae9670ae28e0a9e50d73b2f5676ad105726fb30R307. This is just being repeated here.

Reply:
You are right. My intention was to have only 1 or max 2 base test classes and we endded up with having many of them which increase the complexity. I would in the near future get rid of most of the base test classes and keep only one or 2 at most.